src/Helpers/Utils.js

```diff
@@ -187,14 +187,131 @@ let Utils = {
 	},
 
 
-	encodeQueryString(obj) {
-		return Querystring.stringify(obj);
+	encodeQueryString(obj, deep = false) {
+
+		if (deep) {
+
+
+			//init str
+			let propStrings = [], str = '';
+
+			//convert to simple object
+			if (obj instanceof Observable) obj = obj.toObject();			
+
+			//base url
+			if (obj.baseUrl) str += obj.baseUrl + '?';
+
+			//add props
+			_.each(_.keys(obj), (key) => {
+
+				//skip baseUrl
+				if (key == 'baseUrl') return;
+
+				propStrings = this.__addPropString(propStrings, key, obj[key]);
+
+			});
+			
+
+			//add to querystring
+			str += propStrings.join('&');
+
+			return str;
+
+		} else {
+
+			return Querystring.stringify(obj);
+		}
+
+
+	},
+
+	__addPropString(propStrings, key, value) {
```

Comment:
Kun je dit nog commenten? En in principe gebruiken we 1 _ voor private/interne methods